scipy/stats/_stats.pyx

```diff
@@ -478,4 +478,75 @@ def _local_correlations(distx, disty, global_corr='mgc'):
     corr_mat[local_varx <= 0, :] = 0
     corr_mat[:, local_vary <= 0] = 0
 
-    return corr_mat
\ No newline at end of file
+    return corr_mat
+
+
+ctypedef fused real:
+    float
+    double
+    int
+    long
+
+
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def gaussian_kernel_estimate(points, real[:, :] values, xi, precision):
+    """
+    def gaussian_kernel_estimate(points, real[:, :] values, xi, precision)
```

Comment:
I repeated the signature here because the interpreter doesn't seem able to infer it. E.g. running `help(gaussian_kernel_estimate)` provides

```
gaussian_kernel_estimate(...)
    [docstring here]
```

whereas for a normal python function it would be

```
gaussian_kernel_estimate(points, values, xi, precision)
    [docstring here]
```

I'm happy with either, but find having access to the signature in the docstring useful.

Reply:
This is normal for Cython code, good as is.